src/kem/kyber/ind_cpa.rs

```diff
@@ -48,11 +49,13 @@ fn deserialize_public_key<const K: usize, const T_AS_NTT_ENCODED_SIZE: usize>(
     public_key: &[u8],
 ) -> [PolynomialRingElement; K] {
     let mut t_as_ntt = [PolynomialRingElement::ZERO; K];
-    for (i, t_as_ntt_bytes) in public_key[..T_AS_NTT_ENCODED_SIZE]
```

Comment:
This seems strange. If the function does not need this slice, it does not need to have T_AS_NTT_ENCODED_SIZE as a const parameter either?

Reply:
These were drive-by changes because we don't need the slices. I didn't want to change more. But we indeed don't need the generic anymore here. If that doesn't create too much churn for the proofs, I'm happy to drop the unnecessary const generics.